File: Problem 1/LineReader.java
import java.io.File;
import java.io.FileNotFoundException;
import java.util.ArrayList;
import java.util.Scanner;

public class LineReader {
	static public ArrayList<String> read() {
		ArrayList<String> lines = new ArrayList<String>();
		File fileObj = new File("input.txt");
		
		try {
			Scanner fileReader = new Scanner(fileObj);

			while (fileReader.hasNextLine()) {
				lines.add(fileReader.nextLine());
			}

			fileReader.close();

			return lines;
		} catch (FileNotFoundException e) {
			return null;
		}
	}
}

File: Day 6/Problem 12/LineReader.cs
using System;
using System.Collections.Generic;

namespace Problem_12 {
	class LineReader {
		static public List<String> Read() {
			System.IO.StreamReader file = new System.IO.StreamReader("input.txt");
			List<String> lines = new List<String>();
			String line = file.ReadLine();

			while (line != null) {
				lines.Add(line);
				line = file.ReadLine();
			}

			return lines;
		}
	}
}